bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java

```diff
@@ -85,7 +85,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 return;
             }
             String channelIdWithoutGroup = channelUID.getIdWithoutGroup();
-            int loadpoint = Integer.parseInt(groupId.substring(9));
+            int loadpoint = Integer.parseInt(groupId.substring(9)) + 1;
```

Comment:
Is this fix backwards compatible, or do we need to treat this according to actual version?

Reply:
Is it always possible to upgrade evcc version, or can you imagine any users being stock with older versions? Perhaps it could be asked in the forum, or initially we could assume version 0.111.1 as first version needing this change?

Reply:
I'm relatively sure that version 0.111.1 is the first version needs the change. 
I'm afraid, but I think additional changes are necessary, my environment is still not working correct, problems with Battery SoC, Target SoC and Minimum SoC.

Reply:
Looking at what changed in the last time at evcc, it is `0.111.0` that brings API breaking changes. `0.111.1` is just a small patch release.

> Is it always possible to upgrade evcc version, or can you imagine any users being stock with older versions?

I can’t imagine anyone being stock on older version, because I don’t see a reason why one should. evcc requires only very little resources and I can’t imagine a technical reason why one shouldn’t be able to upgrade.

Reply:
> I'm afraid, but I think additional changes are necessary, my environment is still not working correct, problems with Battery SoC, Target SoC and Minimum SoC.

Should work again now (at least in my tests it works). I'll try to provide an artifact tomorrow.

Reply:
thank you very much, I will test it right away

Reply:
> Should work again now (at least in my tests it works). I'll try to provide an artifact tomorrow.

I assume you are working on that bigger API change?

Reply:
I’ve actually finished work when I wrote that comment, but haven’t had the time (yet) to provide an artifact.

Reply:
@mj6517  I created an artifact for 3.4.x, you can download it from WeTransfer: https://we.tl/t-NP2vaXWFdm.

Reply:
thank's a lot. I've implemented and tested it. Everything works like before. prefect!

Reply:
Thanks for testing!

Reply:
I had a brief look at the code, and it seems you don't have the version information available (currently), so I'll resolve this comment and we'll rely on the documentation asking users to upgrade firmware.